apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -158,7 +158,8 @@ const ReviewApplication = ({ issue, requestIndex }) => {
             {request.review.feedback.length ? request.review.feedback : 'No feedback was provided'}
           </Text.Block>
         </React.Fragment>
-      ) : (
+      )}
+      {!(readOnly || 'review' in request) && (
```

Comment:
I would find this a lot easier to read if we got rid of the parentheses and did something like:

```js
!readOnly && !request.review
```

Reply:
Fixed!